lib/obsolete.gd

```diff
@@ -76,19 +76,29 @@
 ##  <#/GAPDoc>
 ##
 
-BIND_GLOBAL( "DeclareObsoleteSynonym", function( name_obsolete, name_current )
-    local value, orig_value;
+BIND_GLOBAL( "DeclareObsoleteSynonym", function( name_obsolete, name_current, level_arg... )
+    local value, orig_value, printed_warning, level;
```

Comment:
The new argument `level_arg` is not mentioned in the commit message nor the PR description. It seems to be meant to allow overriding the "obsoleteness level" for specific synonyms. I think this is sufficiently non-trivial that at least a brief comment should also be added to the code, perhaps also giving an example of the intended use?

I assume the idea is that over time, certain synonyms are moved to different levels, until they are finally removed (which may never happen in some cases, of course).

If so, shouldn't this argument also be added to `DeclareObsoleteSynonymAttr`?